### ISSUE TITLE
Missing required 'backbone.ckpt' checkpoint?

### ISSUE THREAD

**Issue by brettb** [jhpuah/EEGDM#1]
I'm attempting to evaluate your solution, I am however running into an error for a missing backbone.ckpt when utilizing the pre-trained checkpoint on Huggingface. I've tried creating a dummy file with no luck. If this is a required checkpoint, please upload to Hugginface so I can proceed.

I'm a little confused since I don't see a reference in your code, please let me know if the file is available or if I need to make modifications.

**Comment by jhpuah** [comment]
Hi, very sorry for the late reply, I have uploaded `backbone.ckpt` to Huggingface.

The `backbone.ckpt` is the diffusion model we trained on TUEV training set, and can be obtained from running `pretrain=base`. The `classifier.ckpt` on Huggingface is the finetuned model, trained using rng seed 0. It is standalone and does not require `backbone.ckpt` during inference*. 

Without `backbone.ckpt`, you can check the main result by running `report`, note that `conf/report/base.yaml` config expect 5 checkpoints from different rng, so you will need to override that part:

`python main.py report=base report.checkpoint=["<path to the downloaded checkpoint>"]`

Now with the addition of `backbone.ckpt` on Huggingface, you should be able to start from `cache` or `finetune`. Note that the size of cached TUEV is 94 GB, and 480 GB for CHB-MIT. Caching speed up finetuning significantly but I don't quite remember the exact numbers.


 *The EMA weight of `backbone.ckpt` is actually included in `checkpoint.ckpt`, and the classifier use it to process data during inference if the data is not cached
